tests/bson/bson-binary-serialization_error-004.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+MongoDB\BSON\Binary unserialization requires "data" string and "type" integer fields (__serialize and __unserialize)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_php_version('<', '7.4.0'); ?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+echo throws(function() {
+    unserialize('O:19:"MongoDB\BSON\Binary":1:{s:4:"data";s:6:"foobar";}');
+}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
```

Comment:
I know many of these files don't do this yet, but we should be using the `::class` constant. No need to change this in these files as they're most likely copypasta from the originals, so I'll see if I can do a nice find/replace to take care of these.

Reply:
It would likely require a `sprintf` since we'd also need to capture the string length in the preceding integer. I'll leave this to you to make a ticket if you'd like to track the work.